### PR TITLE
release: version bump to 0.16.3-0.9.0

### PR DESCRIPTION
**Pull request by loloicci**
# Description
It contains

- rust version downgrade (1.55.0 -> 1.54.0)
- change depended cosmwasm to 0.16.3-0.5.1
- fix devtools/setversions

## Types of changes

- [ ] Bug fix (changes which fixes an issue)
- [ ] New feature (changes which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] ETC (build, ci, docs, perf, refactor, style, test)

## Checklist

- [x] I followed the [contributing guidelines](https://github.com/line/wasmvm/blob/main/CONTRIBUTING.md).
- [ ] I have updated the documentation accordingly. (Not needed)
- [ ] I have added tests to cover my changes. (Not needed)
- [x] The PR title and commits are followed [conventional commit form](https://www.conventionalcommits.org/en/v1.0.0).

### CODE REVIEW

**Comment by brew0722** [comment]
libwasmvm.dylib, libwasmvm.so are removed.
we need to commit that the prebuilt binary also

**Review by brew0722 — APPROVED** [review]

**Review by brew0722 — COMMENTED** [review]
sorry, we cannot use the version in the dependencies, is it right? we just use the tag

File: libwasmvm/Cargo.toml [hunk 1]
```diff
@@ -30,8 +30,8 @@ default = []
 backtraces = []
 
 [dependencies]
-cosmwasm-std = { git = "https://github.com/line/cosmwasm", rev = "811d14d8f36dc807c0f8cdf6c36d6ca1d09377b4", features = ["iterator","staking","stargate"] }
-cosmwasm-vm = {  git = "https://github.com/line/cosmwasm", rev = "811d14d8f36dc807c0f8cdf6c36d6ca1d09377b4", features = ["iterator","staking","stargate"] }
+cosmwasm-std = { git = "https://github.com/line/cosmwasm", version = "0.16.3-0.5.1", features = ["iterator","staking","stargate"] }
```
Comment (brew0722) [inline libwasmvm/Cargo.toml:33]: in the previous version, we used the 'tag'
# cosmwasm-std = { git = "https://github.com/line/cosmwasm", tag = "v0.14.0-0.4.0", features = ["iterator","staking","stargate"] }
# cosmwasm-vm = {  git = "https://github.com/line/cosmwasm", tag = "v0.14.0-0.4.0", features = ["iterator","staking","stargate"] }

Comment (loloicci) [reply libwasmvm/Cargo.toml]: Nice finding! Thanks!

**Review by brew0722 — APPROVED** [review]

**Review by shiki-tak — APPROVED** [review]

**Pull request merged**